tools/cmake/toolchains/ARM.cmake

```diff
@@ -64,6 +64,13 @@ function(mbed_set_toolchain_options target)
         )
     endif()
 
+    if(MBED_GENERATE_MAP_TABLE)
```

Comment:
Why is it optional? Shouldn't we always generate the map like the old tools? Was that a requirement?

Reply:
Now I enabled map file generation by default like old tools, please re-review